pkg/kubernetes/client/exec.go

```diff
@@ -9,6 +9,16 @@ import (
 	"strings"
 )
 
+// KubectlCmd returns command a object that will launch kubectl at an appropriate path.
+func KubectlCmd(args ...string) *exec.Cmd {
```

Comment:
```suggestion
func kubectlCmd(args ...string) *exec.Cmd {
```
Does it need to be public?